skimage/measure/fit.py

```diff
@@ -39,6 +44,8 @@ class LineModel(BaseModel):
 
     A minimum number of 2 points is required to solve for the parameters.
 
+    **Deprecated, use ``LineModelND`` instead.**
```

Comment:
A DeprecationWarning should be raised whenever the class is instantiated.  You can find examples elsewhere in the library.